flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/NonDeterministicTests.scala

```diff
@@ -65,11 +70,38 @@ class NonDeterministicTests extends ExpressionTestBase {
   }
 
   @Test
-  def testLocalTimestamp(): Unit = {
-    testAllApis(
-      localTimestamp().isGreater("1970-01-01 00:00:00".toTimestamp),
-      "localTimestamp() > '1970-01-01 00:00:00'.toTimestamp",
-      "LOCALTIMESTAMP > TIMESTAMP '1970-01-01 00:00:00'",
+  def testLocalTimestampInUTC(): Unit = {
+    config.setLocalTimeZone(ZoneId.of("UTC"))
+    val localDateTime = LocalDateTime.now(ZoneId.of("UTC"))
+
+    val formattedLocalTime = localDateTime.toLocalTime.format(DateTimeFormatter.ofPattern("HH:mm:ss"))
+    val formattedLocalDateTime = localDateTime.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))
+
+    // the LOCALTIME/LOCALTIMESTAMP functions are not deterministic, thus we
+    // use following pattern to check it return SQL timestamp in session time zone UTC
+    testSqlApi(
+      s"TIMESUB(LOCALTIME, TIME '$formattedLocalTime') <= 60000",
```

Comment:
TIMESUB?

Reply:
we'd better call it `SUB` rather than `DIFF`  because we assume the t1 produced before t2, if not, we don't know the lag between two inputs in the cornor case that t1 = '23:59:59' and t2 = '00:00:01'